config/locales/views/onboarding/onboarding.yml

```diff
@@ -12,6 +12,82 @@ en:
       update: Update my account
       use_existing_account: Use an existing account
       your_name: Your name
+    completion:
+      new:
+        actions: Actions
+        add_a_meter: Add a meter
+        add_an_inset_day: Add an inset day
+        admin: Admin
+        complete_setup: Complete setup
+        configure_energy_meters: Configure energy meters
+        configure_inset_days: Configure inset days
+        configure_inset_days_message_html: "<p>We've created a calendar for your school based on your school area's term times and national bank holidays.</p><p>Adding your school's inset days helps us make our energy calculations more accurate for days where the school may be closed.</p><p>Once your account is activated, you will be able to add any other dates where your school terms differ from your Local Authority recommended dates.</p>"
+        date: Date
+        day: Day
+        email: Email
+        mpan_mprn: MPAN/MPRN
+        name: Name
+        opening_times: Opening times
+        optional_steps: Optional steps
+        optional_steps_message: It's not necessary for you to fill in the following sections to complete the sign-up process, but the more information you provide the quicker your school can start using Energy Sparks and our energy calculations will become more accurate.
+        password: Password
+        review_and_update_your_answers: Review and update your answers before completing setup
+        role: Role
+        school_meters_empty_message: If you have the details of your energy meters to hand then you can input them here. If not, we can get your meter details from your Local Authority, Multi-academy Trust or school group.
+        section_1:
```

Comment:
I wonder if the sections could have names that match what they're showing, e.g. `.admin_account`, `.school_details`, etc. If we reorganise these as partials, or reorder the page the number might become awkward.